<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(132, 255, 240)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section {
    background-color: rgb(16, 121, 63);
    padding: 2px;
    color: white;
    font-weight: bold;
    font-size: 12px;
}

QTableWidget QTableCornerButton::section {
    background-color: rgb(16, 121, 63);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: white;
	border-radius: 10px;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_data_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: white;
	border-radius: 10px;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: white;
	border-radius: 10px;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Calculate metrics</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: white;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="region_cb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Region name</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="column_num_cb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>3 </string>
    </property>
    <item>
     <property name="text">
      <string>3 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 </string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="draw_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
	background-color: white;
	border-radius: 10px;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>Draw graphic</string>
    </property>
   </widget>
   <widget class="QLabel" name="graphic_lable">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>601</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="po_arabski_ne_ponimayu">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>601</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: white;
	border: 1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>msg  db 'Здесь могла быть ваша реклама', 0x0d, 0x0a, '$</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="min_year_cb">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Min year</string>
    </property>
   </widget>
   <widget class="QComboBox" name="max_year_cb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Max year</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
